src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -304,33 +338,45 @@ module "HostConditions" {
 }
 ```
 
-You have added a New Relic alert channel, subscribed the demo policy to the alert channel, and added a module hosted on Github.
-
-To clone the module from GitHub, run `terragrunt init` and then run `terragrunt apply`. After Terraform is done processing, You will see your Demo Policy now has two conditions in your New Relic account and one alert channel attached.
+You've added a New Relic alert channel, subscribed the demo policy to the alert channel, and added a module hosted on Github.
```

Comment:
Here, you added a New Relic alert channel, subscribed the demo policy to the alert channel, and added a module hosted on Github.